Library/Homebrew/extend/os/mac/info.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+def formula_path
```

Comment:
Sorry to be irritating: general practice is to define the "default" platform (i.e. macOS in this case) in the `cmd/info.rb` and override the methods. In short: you can move the methods from this file into `cmd/info.rb` and delete this file.

Reply:
Thanks, will do! I’ll get the hang of this one day!

Reply:
I tried, but it didn't work, it never got to the Linux overrides.

I reset the branch to how it was for these comments and I'll try again tomorrow.